benchexec/tools/jpf-core.py

```diff
@@ -0,0 +1,60 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2018  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    Tool info for JPF (plain jpf-core)
+    (https://github.com/javapathfinder/jpf-core/).
+    """
+
+    REQUIRED_PATHS = [
+                  "jpf-core/bin",
```

Comment:
I'm not sure what benchexec is using this information for. But afaik, the top-level directory in the verifier archive is stripped off. Hence the path should be `bin` and `build` respectively. Or, if everything in the current zip file is put into an additional directory, then these paths would be correct.